tools/st2-setup-examples

```diff
@@ -23,7 +22,7 @@ echo -e "Bootstrapping virtualenv for ${PACK} pack..."
 if [ -f ${TESTSPATH}/${PACK}/requirements.txt ]; then
     echo "Creating virtualenv for ${PACK}..."
     mkdir -p /opt/stackstorm/virtualenvs
-    virtualenv -p ${PY27} --system-site-packages /opt/stackstorm/virtualenvs/${PACK}
+    virtualenv /opt/stackstorm/virtualenvs/${PACK}
```

Comment:
Looks great, had one query...

On this line - which version of python will this use? Should we have a -p python3, so that on distributions where python2.7 is still the default, that python3 is used? (Or point to /opt/stackstorm/st2/bin/python - which would be python3 on those distributions...)

Similar comment on the st2-setup-tests...